document virtualColumns in native query documentation, fix some redirects (#12917)
apache/druid: docs/querying/searchquery.md

|limit| Defines the maximum number per Historical process (parsed as int) of search results to return. |no (default to 1000)|
|intervals|A JSON Object representing ISO-8601 Intervals. This defines the time ranges to run the query over.|yes|
|searchDimensions|The dimensions to run the search over. Excluding this means the search is run over all dimensions.|no|

> |virtualColumns|Define any [virtual columns](./virtual-columns.md) to be available to use as `searchDimensions`.| no (default none)|

Reviewer: ```suggestion
|virtualColumns|A JSON list of [virtual columns](./virtual-columns.md) available to use in `searchDimensions`.| no (default none)|
```
